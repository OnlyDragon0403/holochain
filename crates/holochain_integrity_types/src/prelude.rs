//! Common types
pub use crate::link::*;
pub use crate::zome::*;
pub use crate::*;
