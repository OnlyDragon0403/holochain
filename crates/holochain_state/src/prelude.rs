pub use crate::chain_sequence::*;
pub use crate::dht_op_integration::*;
pub use crate::element_buf::*;
pub use crate::insert::*;
pub use crate::metadata::*;
pub use crate::query::prelude::*;
pub use crate::source_chain::*;
pub use crate::validation_db::*;
pub use crate::validation_receipts_db::*;
pub use crate::wasm::*;
pub use crate::workspace::*;
pub use crate::*;

pub use holochain_sqlite::prelude::*;

#[cfg(any(test, feature = "test_utils"))]
pub use crate::test_utils::*;
