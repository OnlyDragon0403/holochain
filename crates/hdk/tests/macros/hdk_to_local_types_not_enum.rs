
use hdk::prelude::holochain_deterministic_integrity::prelude::*;

#[hdk_to_local_types]
struct Foo;

fn main() {}
