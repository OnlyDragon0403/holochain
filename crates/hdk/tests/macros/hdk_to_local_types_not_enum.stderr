error: The `to_local_types` macro can only be used on enums.

         = help: Make this an enum.

 --> tests/macros/hdk_to_local_types_not_enum.rs:5:1
  |
5 | struct Foo;
  | ^^^^^^^^^^^
